WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticListenerSubscriber.cs

```diff
@@ -9,7 +10,9 @@
 
     internal sealed class AzureSdkDiagnosticListenerSubscriber : DiagnosticSourceListenerBase<object>, IDisposable
     {
-        public const string DiagnosticListenerName = "Azure.";
+        private const string DiagnosticListenerName = "Azure.";
+        private static readonly HashSet<string> ExcludedSources = new HashSet<string> { "Azure.Cosmos.Request" };
```

Comment:
Do we expect to add other names to this collection?

I don't have a problem with the use of HashSet, but if there's only one name, this could be a string const.

Reply:
great point! changed it to a string constant - I don't envision having more sources and we can always change it if there will ever be a need for it.